release-notes/1.18.2.md

```diff
@@ -5,20 +5,22 @@ The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.18.2.
 Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
 
 ## New features & improvements
-* Android security updates for < insert-month-year > (see Android Security Bulletin - < insert-month-year along with the link to bulletin > for more information).
-* The Android WebView has been updated to < insert-version-along-with-link >.
-
-## Deprecations
-
-< List deprecated features. Provide an alternative, if any, for the users to use instead of the deprecated features >
-
-## Known issues
-
-< List known issues with the current release. Include plans to fix it, if known. >
+* Android security updates for July 2023. See [Android Security Bulletin - July 2023](tba) for more information.
+* The following security patches that were unavailable when Anbox Cloud version 1.18.1 released are included now:
+    *  CVE-2023-21115 A-258834033
+    *  CVE-2023-21121 A-205460459
```

Comment:
```suggestion
```
They were already present, we just weren't sure if they were, this does not need to be called in the release notes.